frontend/containers/website-social-contact/component.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import cx from 'classnames';
+
+import Link from 'next/link';
+
+import Icon from 'components/icon';
+
+import { SOCIAL_DATA } from './constants';
+import type { WebsiteSocialContactProps, SocialType } from './types';
+
+export const WebsiteSocialContact: FC<WebsiteSocialContactProps> = ({
```

Comment:
Some improvements could be made on iPhone 5.

<img src="https://user-images.githubusercontent.com/6073968/158143783-fd5fda88-6cbe-4041-a8a3-478a5c34d218.png" width="320" alt="Component on iPhone 5" />